with-vue-supabase/pages/api/hello.js

```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+export default function handler(req, res) {
```

Comment:
I don't think you need the default `hello` API route if you're not using it in the app